densenet.py

```diff
@@ -613,14 +614,14 @@ def __create_fcn_dense_net(nb_classes, img_input, include_top, nb_dense_block=5,
         x = concatenate([t, skip_list[block_idx]], axis=concat_axis)
 
         # Dont allow the feature map size to grow in upsampling dense blocks
-        _, nb_filter, concat_list = __dense_block(x, nb_layers[nb_dense_block + block_idx + 1], nb_filter=growth_rate,
-                                                  growth_rate=growth_rate, dropout_rate=dropout_rate,
-                                                  weight_decay=weight_decay,
-                                                  return_concat_list=True, grow_nb_filters=False)
+        x_up, nb_filter, concat_list = __dense_block(x, nb_layers[nb_dense_block + block_idx + 1], nb_filter=growth_rate,
+                                                     growth_rate=growth_rate, dropout_rate=dropout_rate,
+                                                     weight_decay=weight_decay,
+                                                     return_concat_list=True, grow_nb_filters=False)
 
     if include_top:
         x = Conv2D(nb_classes, (1, 1), activation='linear', padding='same', kernel_regularizer=l2(weight_decay),
-                   use_bias=False)(x)
+                   use_bias=False)(x_up)
```

Comment:
Cool. That's done.

On the parameters I am seeing 1,127,952 parameters for FC56 instead of 1.5M and 8,167,504 for FC103 instead of 9.4M mentioned on the paper. The weird thing is that the ratio of missing connections is not growing in the same proportion.